doc/start/beginners-guide.rst

```diff
@@ -17,6 +20,9 @@ way of saying "ways of storing data". These storage interfaces include:
 Deep down, though, all three of these are really RADOS object stores. CephFS
 and RBD are just presenting themselves as file systems and block devices.
 
+Storage Manager: What is It?
```

Comment:
You want it all but you can't have it?

As I read this I sort of expect "Storage Manager" to have been used above, but I don't see it there.  So I might suggest taking it out of this heading.

Reply:
This whole section (and indeed, the entire /start directory) is going to be in confusing flux for a while.

I am not opposed to the injection into the documentation of references to any of Mike Patton's musical projects.